Apps/WebClient/src/ClientApp/src/components/modal/verifySMS.vue

```diff
@@ -171,6 +173,21 @@ export default class VerifySMSComponent extends Vue {
             });
     }
 
+    private getTimeout(): number {
+        if (!this.smsResendDateTime) {
+            this.updateSMSResendDateTime({
+                hdid: this.user.hdid,
+                dateTime: new DateWrapper(),
+            });
+        }
+        let resendTime = new Date();
```

Comment:
instead of using Date, lets use DateWrapper

Reply:
changes made

Reply:
dont see the change 😕